docs/Tutorials/GettingStarted.md

```diff
@@ -0,0 +1,403 @@
+3 days in a queue only to be told they have a typo.
+
+The last section of the SpECTRE help text is a description of the highest-level
+(or root) options in the input file. This section starts with the string:
+```
+==== Description of expected options:
+```
+A brief description of the executable follows, and the actual input options are
+listed under the
+```
+Options:
+```
+heading. For example, in the scalar wave executable we see options like
+```
+  AnalyticSolution:
+    Analytic solution used for the initial data and errors
+  DomainCreator:
+    type=DomainCreator
+    The domain to create initially
+```
+
+If you open up the `PlaneWave1DObserveExample.yaml` input file you will find the
+same options listed there, including sub-options for each of them. A reasonable
+way of figuring out what the sub-options of an option are is to cause an input
+file error. For example, to figure out what the sub-options of
+`AnalyticSolution` are I could specify:
+```
+AnalyticSolution:
+  SomeInvalidOptionString:
+```
+If we now run the executable:
+```
+./bin/EvolveScalarWavePlaneWave1D --check-options \
+                                  --input-file ./PlaneWave1DObserveExample.yaml
+```
+we get an error message that will look something like:
+```
+In ./PlaneWave1DObserveExample.yaml:
+In group AnalyticSolution:
+At line 11 column 3:
+Option 'SomeInvalidOptionString' is not a valid option.
+
+==== Description of expected options:
+Analytic solution used for the initial data and errors
+
+Options:
+  PlaneWave:
+    type=PlaneWave
+    Options for the analytic solution
+
+
+############ ERROR ############
+```
+From this we see that the valid options for `AnalyticSolution` are `PlaneWave`
+and nothing else. This method of determining the options by causing an error can
+be applied to any of the options or sub-options.
+
+For now we will ignore most of the options in the input file, but you should
+familiarize yourself with them before running production simulations. The
+choices you make in the input file can have a major impact on how long the
+simulation takes to run and how accurate your result is. Let's focus on the
+`EventsAndTriggers` section. Events and triggers allow specific things to occur
+(the event) when a criteria (the trigger) is met. The syntax is as follows:
+```
+EventsAndTriggers:
+  ? Trigger:
+      OptionsForTrigger
+  : - EventA:
+        OptionsForEventA
+    - EventB:
+        OptionsForEventB
+```
+For example, the trigger and event pairing:
+```
+  ? Slabs:
+      EvenlySpaced:
+        Interval: 3
+        Offset: 5
+  : - ObserveErrorNorms:
+        SubfileName: Errors
+```
+writes the error norms of the evolve variables every Slabs starting on the fifth
+Slab. A Slab in SpECTRE is a collection of time steps. When using global time
+stepping Slabs and time steps are the same. This is not true when using local
+time stepping.
+
+The `ObserveFields` events will write data into the `VolumeFileName` file with
```

Comment:
Explicitly include the snippet from the example yaml with the `ObserveFields` and explain the specific choices as you did with the reduction data. This will also help the reader understand where the h5 group name comes from in the next part of the instruction.